ci.cjs

```diff
@@ -0,0 +1,160 @@
+#! /usr/bin/env node
+
+const fs = require("fs").promises;
+const path = require("path");
+
+function printUsage() {
+  console.log(
+    `Usage: "ci check-consistency" or "ci bump-version %postfix%"`,
+  );
+}
+
+let postfix;
+const mode = process.argv[2];
+
+function validateArgs() {
+  switch (mode) {
+    case "get-version":
+      return true;
```

Comment:
why doesn't it print version?

Reply:
Same story. I don't know, I don't use it :)